Formula/c/capnp.rb

```diff
@@ -24,10 +24,23 @@ class Capnp < Formula
   depends_on "cmake" => :build
 
   def install
```

Comment:
This should fix your `brew linkage` failure:
```suggestion
  depends_on "cmake" => :build
  uses_from_macos "zlib"

  on_linux do
    depends_on "openssl@3"
  end

  def install
```